db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericStatement.java

```diff
@@ -548,6 +548,22 @@ else if (nativeSparkAggregationModeString.equals("forced"))
             }
             cc.setAllowOverflowSensitiveNativeSparkExpressions(allowOverflowSensitiveNativeSparkExpressions);
 
+            String currentTimestampPrecisionString =
+                    PropertyUtil.getCachedDatabaseProperty(lcc, Property.SPLICE_CURRENT_TIMESTAMP_PRECISION);
```

Comment:
I'm not quite sure I understand your suggestion. Do you mean that we should duplicate the parameter?

Reply:
No, I meant either in addition to using a database property (as you are currently doing), or instead of that, we could use a system-level property.  System-level properties get set in places like hbase-site.xml (I don't know the actual .xml file), but if you go into Cloudera Manager you can see all of the splice properties:  Under HBase Configuration search on "splice." and then search down in the results for all properties that start with "splice.".  I was just thinking it might make the setting more visible to use a system-level setting instead of a database property, but I don't want to hold up the build for it.  It's not a big issue.

Reply:
I cannot access system-level parameter from here so that is unfortunately not an option, but that is one more reason to harmonise properties.